### ISSUE TITLE
Issue with single quotes

### ISSUE THREAD

**Issue by tbeuthel** [hexasoftuk/Hexasoft.Zxcvbn#1]
Hi,

I'm experiencing an issue when using single quotes in passwords. the error is the following:

Unexpected token ILLEGAL

Here's the code:

```
var estimator = new ZxcvbnEstimator();
var result = estimator.EstimateStrength("This is a test'");
```

Here's the stack trace:

```
   at Jint.Parser.JavaScriptParser.ThrowError(Token token, String messageFormat, Object[] arguments)
   at Jint.Parser.JavaScriptParser.ScanStringLiteral()
   at Jint.Parser.JavaScriptParser.Advance()
   at Jint.Parser.JavaScriptParser.Lex()
   at Jint.Parser.JavaScriptParser.ParsePrimaryExpression()
   at Jint.Parser.JavaScriptParser.ParseLeftHandSideExpressionAllowCall()
   at Jint.Parser.JavaScriptParser.ParsePostfixExpression()
   at Jint.Parser.JavaScriptParser.ParseUnaryExpression()
   at Jint.Parser.JavaScriptParser.ParseBinaryExpression()
   at Jint.Parser.JavaScriptParser.ParseConditionalExpression()
   at Jint.Parser.JavaScriptParser.ParseAssignmentExpression()
   at Jint.Parser.JavaScriptParser.ParseArguments()
   at Jint.Parser.JavaScriptParser.ParseLeftHandSideExpressionAllowCall()
   at Jint.Parser.JavaScriptParser.ParsePostfixExpression()
   at Jint.Parser.JavaScriptParser.ParseUnaryExpression()
   at Jint.Parser.JavaScriptParser.ParseBinaryExpression()
   at Jint.Parser.JavaScriptParser.ParseConditionalExpression()
   at Jint.Parser.JavaScriptParser.ParseAssignmentExpression()
   at Jint.Parser.JavaScriptParser.ParseExpression()
   at Jint.Parser.JavaScriptParser.ParseStatement()
   at Jint.Parser.JavaScriptParser.ParseSourceElement()
   at Jint.Parser.JavaScriptParser.ParseSourceElements()
   at Jint.Parser.JavaScriptParser.ParseProgram()
   at Jint.Parser.JavaScriptParser.Parse(String code, ParserOptions options)
   at Jint.Engine.Execute(String source)
   at Hexasoft.Zxcvbn.ZxcvbnEstimator.EstimateStrength(String password) in c:\adamszabo\Hexasoft.Zxcvbn\Trunk\Hexasoft.Zxcvbn\Hexasoft.Zxcvbn\ZxcvbnEstimator.cs:line 31
   at ConsoleApplication1.Program.Main(String[] args) in \\Mac\Home\Desktop\zxcvbn\ConsoleApplication1\ConsoleApplication1\Program.cs:line 16
   at System.AppDomain._nExecuteAssembly(RuntimeAssembly assembly, String[] args)
   at System.AppDomain.ExecuteAssembly(String assemblyFile, Evidence assemblySecurity, String[] args)
   at Microsoft.VisualStudio.HostingProcess.HostProc.RunUsersAssembly()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by adam-szabo** [comment]
Hi, thank you very much for reporting, I will take a look.

**Comment by tbeuthel** [comment]
I have gotten around it by escaping the quote for now.. here's the update code:

```
string password = "This is a test'";
var estimator = new ZxcvbnEstimator();
var result = estimator.EstimateStrength(password.Replace("'","\\'"));
```

**Comment by adam-szabo** [comment]
Fixed in 1.0.2. Apologies it took so long to fix.
